test/Driver/emit-interface.swift

```diff
@@ -19,3 +19,21 @@
 // CHECK-EXPLICIT-PATH: swift -frontend
 // CHECK-EXPLICIT-PATH-SAME: emit-interface.swift
 // CHECK-EXPLICIT-PATH-SAME: -emit-parseable-module-interface-path {{.+}}/unrelated.swiftinterface
+
+// Ensure that we emit arguments when we force filelists as well
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-apple-macosx10.9 %s -emit-parseable-module-interface -o %t/foo -module-name foo -force-single-frontend-invocation -driver-filelist-threshold=0 2>&1 | %FileCheck -check-prefix=CHECK-FILELIST %s
+
+// CHECK-FILELIST: swift -frontend
+// CHECK-FILELIST-SAME: -supplementary-output-file-map
+// CHECK-FILELIST-NOT: emit-interface.swift{{ }}
+
+
+// Actually run this and make sure the flags show up in the interface
```

Comment:
This failed because `%t` doesn't exist here, but also you can't run a target-specific test on all platforms. I think you can just move this to another file and use `%target-build-swift` instead.